### DISCUSSION TITLE
How to launch fresh copy of vorta when contributing

### DISCUSSION THREAD

**Discussion by ratchek** [borgbase/vorta#1567] [General]
Hi,

I love this app and I'm looking to contribute. I've followed all the steps in the "contributing guide" and I've installed vorta in a python venv. When I use `which vorta` I'm shown that it indeed the version from my venv being launched (not the copy I have on my system for personal use).

However when the app launches it is still connected to my personal backups as if I launched the system version for backup. The two seem to be coupled. I'm guessing that it's reading the configuration files of the "system" version. Am I missing anything? Is there any default/approved way to make sure that the version of the app I'm working on doesn't have access to those settings and private keys?

**Comment by real-yfprojects** [comment]
Hi,
nice to hear that you want to contribute. We are always looking for help. If you have any question about development and contributing feel free to ask. I can also walk you through solving an issue if desired.

Currently vorta doesn't support a development mode. See #1162. However one might still be useful.
